### ISSUE TITLE
Actions for typified elements

### ISSUE THREAD

**Issue by AlexanderTolmachev** [yandex-qatools/htmlelements#12]
It will be very useful to provide actions for typified elements like <a href="http://selenium.googlecode.com/svn/trunk/docs/api/java/org/openqa/selenium/interactions/Actions.html">WebDriver Actions for WebElements</a>.

**Comment by ham1** [comment]
Any more info on this? The original link is now 404.

**Comment by AlexanderTolmachev** [comment]
Yes, WebDriver repo has been moved. Here is the class I mentioned above: <a href="https://github.com/SeleniumHQ/selenium/blob/c10e8a955883f004452cdde18096d70738397788/java/client/src/org/openqa/selenium/interactions/Actions.java">Actions.java</a>.

**Comment by ham1** [comment]
Is what you were thinking now possible since `TypifiedElement`s now implement `WebElement`? See #121

**Comment by AlexanderTolmachev** [comment]
Yes, you can use WebDriver Actions with TypifiedElements since #121 is done.
But it's not type safe. I mean, you are not preserved from calling something like this:
```
new Actions(driver).sendKeys(button, "some text").perform();
```
